types/outliers/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for outliers 0.0
+// Project: https://github.com/MatthewMueller/outliers
+// Definitions by: Glenn Reyes <https://github.com/glennreyes>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function outliers(input: number[]): number[];
+declare function outliers(path?: string): () => unknown[];
```

Comment:
It looks like this actually returns an `Array.prototype.filter` predicate: `(element: number, index: number, array: number[]) => boolean`.

Reply:
https://github.com/matthewmueller/outliers/blob/9d9725ce75b55018a0b25f93d92538d7ff24b36c/index.js#L26

Reply:
Thanks, will update! 👍